pandas/core/frame.py

```diff
@@ -577,6 +577,12 @@ def _repr_html_(self):
         else:
             return None
 
+    @property
+    def style(self):
+        """ return our styler """
```

Comment:
Shouldn't that get a nicer docstring which explains the purpose of it?

```
Entrypoint to build a styled html representation of the DataFrame.

This can be used to 
* change the overall look of a displayed dataframe
* highlight certain values
* ...
```